### PR TITLE
Added units kvarh and kvar.

### PR DESCRIPTION
**Pull request by Hilbrand**
Also simplified gravity unit.

### CODE REVIEW

**Comment by Hilbrand** [comment]
The build fails in things tests on spotbugs timeout:
```
21:37:20 [INFO] --- sat-plugin:0.6.1:spotbugs (sat-all) @ org.openhab.core.thing.tests ---
21:37:20 [INFO] Adding dependency to findbugs:0.6.1
21:37:20 [INFO] Adding dependency to bug-pattern:1.2.4
21:37:20 [INFO] Adding dependency to spotbugs:3.1.7
21:37:21 [INFO] Fork Value is false
21:37:22      [java] JVM args ignored when same JVM is used.
21:50:55 Timeout: sub-process interrupted
```
```
21:52:18 [ERROR] Failed to execute goal org.openhab.tools.sat:sat-plugin:0.6.1:spotbugs (sat-all) on project org.openhab.core.thing.tests: Unable to execute mojo: Execution null of goal com.github.spotbugs:spotbugs-maven-plugin:3.1.6:spotbugs failed: Timeout: killed the sub-process -> [Help 1]
```
A 'normal' build runs in seconds:
```
18:02:02 [INFO] Adding dependency to spotbugs:3.1.7
18:02:02 [INFO] Fork Value is false
18:02:04      [java] JVM args ignored when same JVM is used.
18:02:28 [INFO] Done SpotBugs Analysis....
```

**Comment by hakan42** [comment]
Out of curiosity, how will those units be used? The description on wikiperia is not really informative for people like me who are not electrical engineers :)

**Comment by Hilbrand** [comment]
Here is an explanation using the :beer:  analogy :smile:  
https://electrical-engineering-portal.com/beer-mug-and-power-factor

These units are used for a new smart meter thing. Where the meter gives those values.

**Comment by Hilbrand** [comment]
@wborn or @cweitkamp can one of you have a look. It is now blocking my dsmr pr and thus blocking migrating the dsmr binding to bnd.

File: bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/library/unit/SmartHomeUnits.java [hunk 1]
```diff
@@ -82,8 +84,8 @@
 
     // Alphabetical ordered by Unit.
     public static final Unit<Acceleration> METRE_PER_SQUARE_SECOND = addUnit(Units.METRE_PER_SQUARE_SECOND);
-    public static final Unit<Acceleration> STANDARD_GRAVITY = new TransformedUnit<>("gₙ",
```
Comment (cweitkamp) [inline bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/library/unit/SmartHomeUnits.java]: Don't we loose the symbol "g" when applying this.

Comment (J-N-K) [reply bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/library/unit/SmartHomeUnits.java]: I think so. Rest looks good IMHO.

**Pull request merged**